wg-data/README.md

```diff
@@ -0,0 +1,33 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/blob/master/generator/README.md
+--->
+# Data Working Group
+
+The WG "Data" is focused on enhancing the support for data/metadata-related tasks within Kubeflow, with a specific focus on the Spark operato and Model Registry. The group aims to simplify and improve data processing between various stages of ML lifecycle. For example, from Data Preparation to model training and fine-tuning. The group also aims to facilitate the ML model's metadata management, while ensuring seamless integration with other Kubeflow components. The goal of Spark on Kubernetes Operator is to simplify the capability of running Apache Spark on Kubernetes. It automates deployment and simplifies lifecycle management of Spark Jobs on Kubernetes. The goal of Model Registry is gather, analyze, and develop model registry requirements of Kubeflow community users.
+
+The [charter](charter.md) defines the scope and governance of the Data Working Group.
+
+## Meetings
+* KF Model Registry community meeting (US/EMEA): [Mondays at 7:00PM-8:00PM Europe/Madrid]() (biweely - every other Monday of the month). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=7:00PM-8:00PM&tz=Europe%2FMadrid).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1DmMhcae081SItH19gSqBpFtPfbkr9dFhSMCgs-JKzNo/edit?usp=sharing).
+
+## Organizers
+
+* Tommy Li (**[@Tomcli](https://github.com/Tomcli)**), IBM
```

Comment:
Tommy is no longer with IBM and not involved.

Reply:
@terrytangyuan @ChenYi015 see f8f544a71670dacdc1baa20c2a28192ebcfba9aa and kindly Resolve if satisfactory.

Reply:
There's no "resolve" button here but this change looks good

Reply:
Here's what I see. I don't see the button for this comment. If you can see it, please help mark as resolved.
![image](https://github.com/user-attachments/assets/ada859ef-18df-4574-836f-e51375f3b5b3)

Reply:
interesting @terrytangyuan i never realized sometimes authors of review could not resolve a comment 🤔 ... i can confirm it shows here as resolved.